mongo/integration/gridfs_test.go

```diff
@@ -369,6 +370,34 @@ func TestGridFS(x *testing.T) {
 			_, err = bucket.OpenDownloadStream(oid)
 			assert.Equal(mt, gridfs.ErrMissingChunkSize, err, "expected error %v, got %v", gridfs.ErrMissingChunkSize, err)
 		})
+		mt.Run("cursor error during read after downloading", func(mt *mtest.T) {
+			// To simulate a cursor error we upload a file larger than the 16MB default batch size,
+			// so Read() will actually execute a getMore against the server. Since the ReadDeadline is
```

Comment:
IIUC the test itself won't send a `getMore`. It should fail on the first iteration of the cursor.
```suggestion
			// so the underlying cursor remains open on the server. Since the ReadDeadline is
```

Reply:
Ah you're right. Previously I was using a `failpoint` on `getMore`, but the timeout error just comes from iterating the cursor. Changed.